pkg/runtime/executor.go

```diff
@@ -307,6 +323,26 @@ func (e *Executor) findViolations(results *Output) error {
 	return nil
 }
 
+// getVulnerabilities add vulnerability findings to output
+func (e *Executor) getVulnerabilities(results *Output, options map[string]interface{}) output.AllResourceConfigs {
+
+	res := e.vulnerabilityEngine.FindVulnerabilities(results.ResourceConfig, options)
+
+	return res
```

Comment:
can be modified to `return e.vulnerabilityEngine.FindVulnerabilities(results.ResourceConfig, options)`